js/_codemirror.ts

```diff
@@ -50,11 +50,11 @@ const asmErrorTooltip = {
     'color':      'var(--background)',
 };
 
-const fontFamily = "'Source Code Pro', monospace";
+const fontFamily = '\'Source Code Pro\', monospace';
 
 export const extensions = {
     // Extensions.
-    base: [
+    'base': [
```

Comment:
This change looks unrelated to TS

Reply:
Removing it gives
```
  57:5  error  Inconsistently quoted property 'base' found             quote-props
  73:5  error  Inconsistently quoted property 'bracketMatching' found  quote-props
  74:5  error  Inconsistently quoted property 'dark' found             quote-props
```

We have the rule `  quote-props: [ error, consistent-as-needed ]`, and [eslint quote-props docs](https://eslint.org/docs/rules/quote-props) notes:

> "consistent-as-needed" requires quotes around all object literal property names if any name strictly requires quotes, otherwise disallows quotes around object property names

Quotes are required here because `c-sharp` and `f-sharp` on that object require quotes. I don't know why the linter didn't catch this before.

Reply:
Ah it was because it was previously ignored in the `.eslintrc.yml`, yours fired because you changed the extension. I can't remember why I ignored it, maybe it was from a far more aggressive eslint policy and it was far too noisy. Either way I've fixed that, applied the property quoting, and added the `avoidEscape` rule to `quotes` in 45afd88